<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationWidget</class>
 <widget class="QWidget" name="CommunicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>1359</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Communication</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QTreeWidget" name="sessionView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="trialView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="performerView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="localView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="workspace">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionUpdate">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>update data</string>
   </property>
  </action>
  <action name="actionPerformer_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Performer</string>
   </property>
   <property name="toolTip">
    <string>change view</string>
   </property>
  </action>
  <action name="actionSession_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Session</string>
   </property>
   <property name="toolTip">
    <string>change view</string>
   </property>
  </action>
  <action name="actionTrial_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Trial</string>
   </property>
   <property name="toolTip">
    <string>change view</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>download server trial</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLoad_trial">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load trial</string>
   </property>
   <property name="toolTip">
    <string>load local trial</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbort_download">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abort download</string>
   </property>
   <property name="toolTip">
    <string>abort downloading server trial</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add all elements</string>
   </property>
   <property name="toolTip">
    <string>Add all files to workspace</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove all</string>
   </property>
   <property name="toolTip">
    <string>Remove all files from workspace</string>
   </property>
  </action>
  <action name="actionAdd_GRF">
   <property name="text">
    <string>Add GRF</string>
   </property>
   <property name="toolTip">
    <string>Add GRF</string>
   </property>
  </action>
  <action name="actionAdd_videos">
   <property name="text">
    <string>Add videos</string>
   </property>
   <property name="toolTip">
    <string>Add video files</string>
   </property>
  </action>
  <action name="actionAdd_EMG">
   <property name="text">
    <string>Add EMG</string>
   </property>
   <property name="toolTip">
    <string>Add EMG</string>
   </property>
  </action>
  <action name="actionRemove_GRF">
   <property name="text">
    <string>Remove GRF</string>
   </property>
   <property name="toolTip">
    <string>Remove GRF</string>
   </property>
  </action>
  <action name="actionRemove_videos">
   <property name="text">
    <string>Remove videos</string>
   </property>
   <property name="toolTip">
    <string>Remove video files</string>
   </property>
  </action>
  <action name="actionRemove_EMG">
   <property name="text">
    <string>Remove EMG</string>
   </property>
   <property name="toolTip">
    <string>Remove EMG</string>
   </property>
  </action>
  <action name="actionLocal_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local</string>
   </property>
   <property name="toolTip">
    <string>change view</string>
   </property>
  </action>
  <action name="actionAdd_MOCAP">
   <property name="text">
    <string>Add MOCAP</string>
   </property>
   <property name="toolTip">
    <string>Add MOCAP</string>
   </property>
  </action>
  <action name="actionRemove_MOCAP">
   <property name="text">
    <string>Remove MOCAP</string>
   </property>
   <property name="toolTip">
    <string>Remove MOCAP</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbort_download</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>abortPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDownload</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>downloadPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_trial</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>loadPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPerformer_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>performerViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSession_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>sessionViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTrial_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>updatePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CommunicationWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>contextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>addToWorkspace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>removeFromWorkspace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workspace</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>contextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessionView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trialView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>789</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>performerView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>861</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>861</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLocal_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>localViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_EMG</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>addEMG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_GRF</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>addGRF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_MOCAP</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>addMocap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_videos</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>addVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_EMG</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>removeEMG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_GRF</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>removeGRF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_MOCAP</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>removeMocap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_videos</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>removeVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>performerViewPressed(bool)</slot>
  <slot>sessionViewPressed(bool)</slot>
  <slot>trialViewPressed(bool)</slot>
  <slot>updatePressed()</slot>
  <slot>downloadPressed()</slot>
  <slot>abortPressed()</slot>
  <slot>contextMenu(QPoint)</slot>
  <slot>loadPressed()</slot>
  <slot>trialContextMenu(QPoint)</slot>
  <slot>addToWorkspace()</slot>
  <slot>removeFromWorkspace()</slot>
  <slot>localViewPressed(bool)</slot>
  <slot>addEMG()</slot>
  <slot>removeEMG()</slot>
  <slot>addGRF()</slot>
  <slot>removeGRF()</slot>
  <slot>addVideo()</slot>
  <slot>removeVideo()</slot>
  <slot>addMocap()</slot>
  <slot>removeMocap()</slot>
 </slots>
</ui>
